aten/src/ATen/native/quantized/IndexKernel.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <ATen/native/DispatchStub.h>
```

Comment:
why?

Reply:
Well, as I comment, it's because that during an implementation of some operators for `privateuse1` backend, If this header is not included, we can't use `masked_fill_kernel_quantized_stub`, it would throw an error during compilation.